### PR TITLE
add new company - Accor

### PR DESCRIPTION
**Pull request by dhruvchandak30**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by dhruvchandak30** [comment]
@KieranRobson  Can you please add `hacktoberfest` and `hacktoberfest-accepted` labels.
It would be a great help for me.

File: company-profiles/accor.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+16,156 employees on LinkedIn
+
+## Remote status
+
+The team is remote and in person, located in France and 110 other countries.
+
+## Region
+
+France, Issy-les-Moulineaux
```
Comment (dougaitken) [inline company-profiles/accor.md:17]: Region is where the company hires - the section above lists 110 countries so I'd recommend using worldwide if they hire remotely in those countries

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
@dhruvchandak30 The repository is labelled hacktoberfest so you'll get the participation.

Can you check my review please?

**Comment by dhruvchandak30** [comment]
> @dhruvchandak30 The repository is labelled hacktoberfest so you'll get the participation.
> 
> Can you check my review please?

Yes, Done with the changes

**Review by dougaitken — APPROVED** [review]

**Comment by dhruvchandak30** [comment]
@dougaitken Can you please merge this PR, as I have pushed from my main branch, I am not able to make more PR (I am but it is also showing these two commits).

**Comment by dougaitken** [comment]
> as I have pushed from my main branch, I am not able to make more PR (I am but it is also showing these two commits).

best to use branches for new features rather than `main` 😉

**Pull request merged**